File: tests/fixtures/sample.java
package com.example;

public class Calculator {
    private int value;

    public Calculator(int initial) {
        this.value = initial;
    }

    public int add(int x) {
        this.value += x;
        return this.value;
    }

    private void reset() {
        this.value = 0;
    }
}

interface Drawable {
    void draw();
    String getName();
}

enum Status {
    ACTIVE,
    INACTIVE,
    PENDING
}

class Helper {
    static int multiply(int a, int b) {
        return a * b;
    }
}

File: tests/fixtures/sample.cs
using System;

public class Calculator
{
    private int _value;

    public Calculator(int initial)
    {
        _value = initial;
    }

    public int Add(int x)
    {
        _value += x;
        return _value;
    }

    private void Reset()
    {
        _value = 0;
    }
}

public interface IDrawable
{
    void Draw();
    string GetName();
}

public enum Status
{
    Active,
    Inactive,
    Pending
}

public struct Point
{
    public int X;
    public int Y;
}

public record User(string Name, int Age);
